content/linked-terms.js

```diff
@@ -22,5 +22,9 @@ module.exports = [
   {
     matches: 'dvc experiments',
     url: '/doc/command-reference/exp'
+  },
+  {
+    matches: 'DVCLive',
+    url: '/doc/dvclive'
```

Comment:
This is a good idea but I'm not sure we always want to use `DVCLive` (code quote) or for it to go to /doc/dvclive always. This feature is meant for command and API function names, basically.

Reply:
Got it! I misunderstood your comment in the other P.R.